Directory.Build.props

```diff
@@ -111,6 +111,7 @@
   <PropertyGroup>
     <!-- Ensure API docs are available. -->
     <NoWarn>$(NoWarn.Replace('1591', ''))</NoWarn>
+    <NoWarn>$(NoWarn),0105</NoWarn>
```

Comment:
I don't see these warnings in rolling builds of 'main'. Could this be conditional on `'$(BuildingInsideVisualStudio)' == 'true'`❔

Also concerned about doing this everywhere. How many projects or (maybe better) test asset directories need this suppression❔

Reply:
These are semi-colon delimited no?

Reply:
> Also concerned about doing this everywhere. How many projects or (maybe better) test asset directories need this suppression❔

Just in middleware.snlf I get 174 errors across 25 projects, most of which are product assemblies.